### PR TITLE
refactor: switch LoadingMembersList to styled components

### PR DESCRIPTION
**Pull request by Chandrasekar-G**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | iPhone 6 (iOS 10.3) |
| Bug fix?         | no      |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  | #532        |

---

## Screenshots

| Before   | After    |
| -------- | -------- |
|before|after|

## Description

Migrated LoadingMembersList to styled-components

### CODE REVIEW

File: src/components/loading-indicators/loading-members-list.component.js [hunk 1]
```diff
@@ -47,20 +49,20 @@ export class LoadingMembersList extends Component {
...
 
-        <View style={styles.avatarContainer}>
+        <AvatarContainer>
           {[...Array(10)].map((item, index) => {
             return (
-              <Animated.View
+              <Avatar
                 key={index} // eslint-disable-line react/no-array-index-key
-                style={[styles.avatar, { opacity: this.state.fadeAnimValue }]}
+                style={{ opacity: this.state.fadeAnimValue }}
```
Comment (chinesedfan) [inline src/components/loading-indicators/loading-members-list.component.js]: How about designing `opacity` as a prop of `Avatar`? Then,
```
const Avatar = styled(Animated.View)`
    opacity: ${prop => prop.opacity};
    // blabla
`
```

Comment (Chandrasekar-G) [reply src/components/loading-indicators/loading-members-list.component.js]: @chinesedfan  I pass the prop from the parent component 
`<LoadingMembersList title={t('MEMBERS', locale)} opacity={new Animated.Value(0)}/>`

I use it in Avatar using props
`opacity: ${props => props.opacity};`

I'm able to get the props in the constructor and it is of type _AnimatedValue_

But inside the styled component the props seems to be undefined and hence I get the below exception in the device. Any idea why?

simulator screen shot 26-oct-2018 1 03 26 pm

Comment (Chandrasekar-G) [reply src/components/loading-indicators/loading-members-list.component.js]: @chinesedfan any other changes required here?

Comment (chinesedfan) [reply src/components/loading-indicators/loading-members-list.component.js]: The space between section CONTRIBUTORS and section SOURCE are larger before. Is it normal?

Comment (Chandrasekar-G) [reply src/components/loading-indicators/loading-members-list.component.js]: I tried reverting to master and took a screenshot again. 
before

Not sure what was causing the extra space in the first place. The spacing right now is what is present in the master.

Comment (chinesedfan) [reply src/components/loading-indicators/loading-members-list.component.js]: At first, 

- with master codes, it was always larger space
- with your codes, it was smaller space at the first render. And if visited the second repository, it became larger space

Then I removed all `node_modules` and `yarn` again. It is always larger space, no matter with master or your codes. Well, it's no a big problem, so don't mind about it.

**Pull request merged**